src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMapping.java

```diff
@@ -49,6 +55,26 @@ public static HearingChannel getHearingChannel(@Valid SscsCaseData caseData) {
         }
     }
 
+    public static HearingChannel getHearingChannel(@Valid SscsCaseData caseData, ReferenceDataServiceHolder referenceDataServiceHolder) {
+        log.info("Get the next hearing channel {}", referenceDataServiceHolder.isAdjournmentFlagEnabled());
+        log.info("Adjournment next hearing channel {}", caseData.getAdjournment().getTypeOfNextHearing());
+        if (referenceDataServiceHolder.isAdjournmentFlagEnabled() && nonNull(caseData.getAdjournment().getTypeOfNextHearing())) {
```

Comment:
Should this also have isYes(getAdjournmentInProgress())?
```suggestion
        if (referenceDataServiceHolder.isAdjournmentFlagEnabled() && isYes(getAdjournmentInProgress()) && nonNull(caseData.getAdjournment().getTypeOfNextHearing())) {
```

Reply:
Updated as per the suggestion and added the test case as well.